engine/include/cubos/engine/render/shadows/caster.hpp

```diff
@@ -0,0 +1,25 @@
+/// @file
+/// @brief Component @ref cubos::engine::ShadowCaster.
+/// @ingroup render-shadows-plugin
+
+#pragma once
+
+#include <glm/vec2.hpp>
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Component which contains the options common to all shadow casters.
+    /// @ingroup render-shadows-plugin
+    struct CUBOS_ENGINE_API ShadowCaster
+    {
+        CUBOS_REFLECT;
+
+        float bias;               ///< Shadow bias.
+        float blurRadius;         ///< Shadow blur radius.
+        glm::uvec2 shadowMapSize; ///< Size of the shadow map, in pixels.
```

Comment:
Hm, from our discussion yesterday regarding the shadow atlas, ig this no longer makes sense.
Maybe just omit it for now? Maybe later on we can assign some kind of 'priority', where casters with higher priorities get bigger texture slots or something.

Reply:
Yeah, you're right, it makes more sense to think of these details when the shadow atlas is implemented.